plugin/trino-memsql/src/main/java/io/trino/plugin/memsql/MemSqlClient.java

```diff
@@ -143,12 +140,11 @@ protected boolean filterSchema(String schemaName)
         }
 
         switch (typeHandle.getJdbcType()) {
+            case Types.CHAR:
+            case Types.NCHAR:
```

Comment:
Does memsql use NCHAR? (mysql doesn't seem to)

Reply:
The default mappings are generic and need to cover all options. For memsql specific mappings, we whould use only the things we observe & test.

if you keep it, add explicit comment that it's a dummy

Reply:
I would prefer to remove it, but I am scared of introducing a regression.

Reply:
> if you keep it, add explicit comment that it's a dummy